envoy/datadog_checks/envoy/envoy.py

```diff
@@ -136,8 +136,8 @@ def check(self, _):
 
             # If the value isn't an integer assume it's pre-computed histogram data.
             except (ValueError, TypeError):
-                for metric, value in parse_histogram(metric, value):
-                    self.gauge(metric, value, tags=tags)
+                for histo_metric, histo_value in parse_histogram(metric, value):
```

Comment:
why this change? seems unrelated

Reply:
Because we are getting this eror: `B020 Found for loop that reassigns the iterable it is iterating with each iterable value`